src/osg_display/gracc_datasource.py

```diff
@@ -83,9 +83,8 @@ def disconnect(self):
     def connect_gracc_url(self, gracc_url):
         try:
             self.es = elasticsearch.Elasticsearch(
-                [gracc_url], timeout=300, use_ssl=True, verify_certs=True,
-                ca_certs='/etc/ssl/certs/ca-bundle.crt')
```

Comment:
So, just to confirm, newer ES libs work without `ca_certs` ?

Reply:
Testing seemed fine without the `ca_certs` path. The path was specific to EL, and didn't exist on Ubuntu. (Which I ended using for the container. I don't have any strong opinions on OS, if you prefer a different base image.)